translate/storage/csvl10n.py

```diff
@@ -416,12 +415,11 @@ def serialize(self):
         return source.encode(self.encoding)
 
     def getoutput(self):
-        outputfile = BytesIO()
-        writer = csv.DictWriter(outputfile, self.fieldnames, extrasaction='ignore', dialect=self.dialect)
-        # write header
-        hdict = dict(map(None, self.fieldnames, self.fieldnames))
-        writer.writerow(hdict)
+        output = csv.StringIO()
+        writer = csv_utils.UnicodeDictWriter(
+            output, self.fieldnames, encoding=self.encoding, extrasaction='ignore',
+            dialect=self.dialect)
```

Comment:
Sure. But lets try to align with our styleguide: http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#continuing-long-statements